### PR TITLE
document how to delete a JWT credential for a consumer

### PR DESCRIPTION
**Pull request by therealgambo**
As the title suggest, documenting how to delete a JWT credential for a consumer.

### CODE REVIEW

**Comment by Tieske** [comment]
on formatting: both are path parameters, then why list one as path parameter the other as query parameter?

On a side note: this doc could use a table of contents

**Comment by therealgambo** [comment]
You're right @Tieske, I've removed the table and displayed it like the other param.

**Comment by Tieske** [comment]
@therealgambo thank you!

File: app/plugins/jwt.md [hunk 1]
```diff
@@ -108,6 +109,18 @@ form parameter                 | default         | description
 `rsa_public_key`<br>*optional* |                 | If `algorithm` is `RS256` or `ES256`, the public key (in PEM format) to use to verify the token's signature.
 `secret`<br>*optional*         |                 | If `algorithm` is `HS256` or `ES256`, the secret used to sign JWTs for this credential. If left out, will be auto-generated.
 
+### Delete a JWT credential
+
+You can remove a Consumers JWT credential by issuing the following HTTP request:
```
Comment (thibaultcha) [inline app/plugins/jwt.md]: "a Consumer's"

File: app/plugins/jwt.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ nav:
     items:
       - label: Create a Consumer
       - label: Create a JWT credential
+      - laebl: Delete a JWT credential
```
Comment (thibaultcha) [inline app/plugins/jwt.md]: typo: label

File: app/plugins/jwt.md [hunk 3]
```diff
@@ -108,6 +109,18 @@ form parameter                 | default         | description
...
+
+You can remove a Consumers JWT credential by issuing the following HTTP request:
+
+```bash
+$ curl -X DELETE http://kong:8001/consumers/{consumer}/jwt/{id}
+HTTP/1.1 204 No Content
+```
+
+`consumer`: The `id` or `username` property of the [Consumer][consumer-object] entity to associate the credentials to.
+`id`: The `id` of the JWT credential.
```
Comment (thibaultcha) [inline app/plugins/jwt.md]: This won't be parsed as a list by the markdown parser and will be on the same line.

**Pull request merged**